unit_test_candidates.md

```
@@ -0,0 +1,140 @@
+# Unit Test Candidates for DrevOps Behat Steps
+
+This document identifies methods and code sections within the DrevOps Behat Steps traits that are good candidates for unit testing.
+
+## Selection Criteria
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)